react-rust-postgres/.docker/docker-compose.yaml

```diff
@@ -0,0 +1,54 @@
+name: react-rust-postgres
+services:
+  frontend:
+    build:
+      context: ../frontend
+      target: dev-envs
+    networks:
+      - client-side
+    ports:
+      - 3000:3000
+    volumes:
+      - ./frontend/src:/code/src:ro
+      - /var/run/docker.sock:/var/run/docker.sock
+
+  backend:
+    build:
+      context: ../backend
+      target: dev-envs
+    environment:
+      - RUST_LOG=debug
+      - PG_DBNAME=postgres
+      - PG_HOST=db
+      - PG_USER=postgres
+      - PG_PASSWORD=mysecretpassword
+      - ADDRESS=0.0.0.0:8000
+    networks:
+      - client-side
+      - server-side
+    volumes:
+      - ./backend/src:/code/src
+      - backend-cache:/code/target
```

Comment:
Same - I think these can both go

(Cache volumes that live longer than dev env for e.g. packages should def be on our roadmap tho 😉)